### ISSUE TITLE
Trouble with Android connectivity check

### ISSUE THREAD

**Issue by real-or-random** [DigitaleGesellschaft/DNS-Resolver#20]
Some Android devices connected to a router configured to use this service sporadicly believe they don't have an Internet connection.  This happens since a few days and switching to a different DoT service works.

I'm not exactly sure what's going on. It's somewhat difficult to debug this since I don't have a command line resolver on Android yet. But I saw at least one instance of Chrome on Android telling me that it received a NXDOMAIN for connectivitycheck.gstatic.com .

**Comment by ryru** [comment]
Hi @real-or-random 

To clarify, you configured DoT on your Android phone (Network & Internet > Advanced > Private DNS), correct? Do you have multiple devices with this behaviour?

Could you provide these information or double check this:

1. Do you face this problem with WiFi and mobile data?
2. Does this happen with both our Servers (`dns1.digitale-gesellschaft.ch` and `dns2.digitale-gesellschaft.ch`)?
3. Can you reproduce this with kdig (`kdig +tls @185.95.218.42 -tA connectivitycheck.gstatic.com`)?

**Comment by real-or-random** [comment]
1. No, I haven't configured Private DNS in Android. I encountered this on multiple Android devices that use my home router for resolving. The router is configured to use `dns.digitale-gesellschaft.ch`. Of course the issue could be the router but as I said, it was working fine earlier. I didn't change anything in the router. And configuring it to use   `1dot1dot1dot1.cloudflare-dns.com` immediately resolved the issue while changing back to   `dns.digitale-gesellschaft.ch` brought it back.

2. When typing this, I can't reproduce it with any of the two. But I kept `dns2` now for testing, let's see.

I was facing this for example in the morning and when I opened this issue.

3. Unfortunately I was never able to reproduce this on the command line, neither via the router nor when querying your servers directly...
